effectsize.Rmd

```diff
@@ -0,0 +1,99 @@
+# Effect size {#effectsize}
+<div style="text-align:center">
+```{r}
+knitr::include_graphics("figures/effectsize/teaser.png")
```

Comment:
Off the top of your head, do you have a code/pointer for placing two ggplot objects in one figure with a specific offset?